src/MonoMod.RuntimeDetour/DetourContext.cs

```diff
@@ -77,6 +77,7 @@ private sealed class ContextScopeHandler : ScopeHandlerBase {
         /// which can be used in a <see langword="using"/> block to automatically pop it from the context stack.
         /// </summary>
         /// <returns>A <see cref="DataScope"/> which manages the lifetime of this context on the context stack.</returns>
+        [CLSCompliant(false)] //TODO: DataScope isn't CLS compliant - is this bad?
         public DataScope Use() => PushContext(this);
```

Comment:
I left this one because I do intend to go through Utils and annotate CLS compliance at some point, I just haven't gotten around to it yet.

Reply:
I'll remove the TODO + annotation in that case